codes/quantum/qubits/quantum_triorthogonal.yml

```diff
@@ -0,0 +1,46 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: quantum_triorthogonal
+physical: qubits
+logical: qubits
+
+name: '(Quantum) Triorthogonal'
+
+introduced: '\cite{arXiv:1209.2426}'
+
+description: 'A triorthogonal \(m \times n\) binary matrix is triorthogonal if its rows \(r_1, \ldots, r_m\) satisfy \(|r_i \cdot r_j| = 0\) and \(|f_i \cdot f_j \cdot f_k| = 0\), where addition and multiplication are done on \(\mathbb{Z}_2\). The triorthogonal code associated with the matrix is constructed by mapping non-zero entries in even-weight rows to X operators, and Z operators for each row in the orthogonal complement.'
+
+protection: 'Protects against distance 2 Pauli errors \cite{arXiv:1209.2426}'
```

Comment:
You meant weight two? Is the distance always 3? If so, then add first sentence in description that says this is a family of [[n,k,3]] CSS codes constructed using triorthogonal matrices.